ow_power_system/src/PredictionHandler.cpp

```diff
@@ -14,7 +16,7 @@ using namespace PCOE;
 
 // The indices use to access temperature information.
 // This might change to median SOC or RUL index or fixed percentile.
-//
+// (predecessor comment to SU22, unsure exactly what it means)
```

Comment:
I don't know what the preceding comments mean, either, but what does SU22 mean?

Reply:
'SU22' stands for summer 2022. I figured it was best to date comments like that for the future. I'll go back and remove the abbreviation tomorrow morning though, it's not particularly necessary here.